### DISCUSSION TITLE
[Vote ended on 2022-10-26] Clarify SCM and release requirements

### DISCUSSION THREAD

**Discussion by mariolenz** [ansible-community/community-topics#153] [Votes done]
Issue:

- https://github.com/ansible-community/community-topics/issues/148

PR with new wording:

- https://github.com/ansible-collections/overview/pull/218

Vote `+1` to merge https://github.com/ansible-collections/overview/pull/218 or `-1` not to merge

**Comment by mariolenz** [comment]
+1

**Comment by gotmax23** [comment]
+1

**Comment by briantist** [comment]
+1

**Comment by felixfontein** [comment]
+1

**Comment by Andersson007** [comment]
+1 (great job, thanks for the clarifications)

**Comment by markuman** [comment]
+1

**Comment by russoz** [comment]
+1

**Comment by acozine** [comment]
+1
